.github/workflows/test.yaml

```diff
@@ -36,6 +36,10 @@ jobs:
                       dependencies: lowest
                       allowed-to-fail: false
                       variant: normal
+                    - php-version: '8.0'
```

Comment:
Shouldn't these changes be handled by dev-kit?

Reply:
https://github.com/sonata-project/dev-kit/blob/master/config/projects.yaml#L401-L412